frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/ModelManager.java

```diff
@@ -76,7 +79,8 @@ public ModelArchive registerModel(String url, String defaultModelName)
                 1,
                 100,
                 configManager.getDefaultResponseTimeout(),
-                defaultModelName);
+                defaultModelName,
+                preforkInit);
```

Comment:
Do you want to call this preloadModel? Some places call it preforkInit and some places call it preloadModel.

Reply:
changed to preloadModel